backend/spec/system/backoffice/projects_spec.rb

```diff
@@ -0,0 +1,41 @@
+require "system_helper"
+
+RSpec.describe "Backoffice: Projects", type: :system do
+  let(:admin) { create(:admin, email: "admin@example.com", password: "SuperSecret6", first_name: "Admin", last_name: "Example") }
+  let!(:project) {
+    project = create(
+      :project,
+      name: "Project ultra name",
+      project_developer: create(:project_developer, account: create(:account, name: "Ultra project developer name")),
+      trusted: true
+    )
+    project.update(priority_landscape: create(:location, location_type: "region", name: "Test mosaic"))
```

Comment:
haha, this is a bit hacky, I would prefer if you can define geometries for project and this location (they can be same so you are sure they intersect) ;)